src/Tabs.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TabsListTab
+ *
+ * Direct child of `TabsList`.
+ * Renders individual tab control.
+ * Accepts a `selected` prop.
+ */
+export const TabsListTab = React.createClass({
```

Comment:
That's strange - we definitely have custom methods on [components in mup-web](https://github.com/meetup/mup-web/blob/master/src/features/find/FindContainer.jsx#L28) so I'll try to dig in to this.

Not sure if you tried it exactly as your example code indicates, but `render` does have to return a React element (usually with JSX), so there's an off chance that's the problem